tests/Backend/Snowflake/WhereFilterTest.php

```diff
@@ -170,6 +170,28 @@ public function testExportTableInvalidComparingOperator()
         $this->getExportedTable($tableId, ['whereFilters' => $where]);
     }
 
+    public function testInvalidStructuredQuery()
```

Comment:
tohle mi lokálně taky padá "Argument 1 passed to Storage_Service_ExportWhereFilter::fromArray() must be of the type array, string given, called in \/var\/www\/html\/application\/modules\/storage\/services\/BucketBackend\/TableExportOptions.php on line 389"

Reply:
a určitě sis checkoutnul branch, kterou to testuje ?

Reply:
Jo, asi nějak jo. Ale nevim, jak k tomu došlo. Protože jsem ji checkoutnul. Jediné co mě napadná je nějaký glitch v apachi nebo mi ji něco odcheckoutnulo na pozadí... teď jsem mezitím dělal jinde, tak nevím. Ale ten horní test mi padal s novou error message (tj. běželo nové connection) a chyba je, že to očekává starou error message.

Reply:
Teď už mi to proběhlo. Tohle. To horní furt ne.